src/OpenLoco/src/Map/RoadElement.cpp

```diff
@@ -18,7 +18,7 @@ namespace OpenLoco::World
         if (owner() == CompanyId::neutral || CompanyManager::isPlayerCompany(owner()))
             return true;
 
-        if (!(getGameState().roadObjectIdIsTram & (1 << roadObjectId())))
+        if (!(getGameState().roadObjectIdIsNotTram & (1 << roadObjectId())))
             return true;
 
         if (sequenceIndex())
```

Comment:
below here on L27 it checks for mods but i think that is incorrect. The L21 means trams are not included which means mods do not exist!